specs/MultiProfile.md

```diff
@@ -328,6 +362,13 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// Delete the profile completely. All webviews on this profile will be closed and  
+  /// the profile directory on disk will be deleted.
```

Comment:
We'll need to document what happens to Profile objects and CookieManager objects and whatever else objects that are associated with a deleted Profile. The Profile object's methods and properties should probably all throw? I guess it should match the behavior of a closed CoreWebView2.